targets/GenApi.targets

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
For now: 👍. Long-term, I would expect this to be accomplished by a conditional NuGet package reference.